### ISSUE TITLE
stack search

### ISSUE THREAD

**Issue by flip111** [commercialhaskell/stack#2772]
It would be nice if it were possible to use `stack search` to search through all available packages, similar to `apt-cache search` on ubuntu.

**Comment by lwm** [comment]
Yeah, maybe with a similar output to `stack templates` (with synopsis string).
